src/components/bordered-info-box/BorderedInfoBox.tsx

```diff
@@ -45,10 +45,10 @@ export const BorderedInfoBox = ({
   );
 };
 
-const useStyles = (type: BorderedInfoBoxProps['type'], textColor: string) =>
-  StyleSheet.createThemeHook((theme) => ({
+const useStyles = (type: BorderedInfoBoxProps['type'], textColor: string) => {
+  return StyleSheet.createThemeHook((theme) => ({
     container: {
-      borderColor: addOpacity(textColor, 0.1),
+      borderColor: addOpacity(textColor, 0.2),
```

Comment:
seems like the only thing that changes was from `0.1` to `0.2` we might then revert all the other things as before, here now seems like we added a return and changed the position of the import.